CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+v1.6.0
+------
+* **Release date:** 2018-12-21.
+* **Requires:** [tileserver v2.2.0](https://github.com/tilezen/tileserver/releases/tag/v2.2.0) and [tilequeue v2.2.1](https://github.com/tilezen/tilequeue/releases/tag/v2.2.1) and [rawr_tiles v1.0.0](https://github.com/tilezen/raw_tiles/releases/tag/v1.0.0) and [coanacatl v1.0.0](https://github.com/tilezen/coanacatl/releases/tag/v1.0.0).
+
+  #### ENHANCEMENTS
+
+  * **Significant file size reductions** of ~30% globally by additional geometry simplification, dropping features, dropping properties, and more aggressive merging to multi-lines and multi-polygons in low- and mid-zooms. _Chart shows sizes in bytes (logarithmic scale), based on top 100,000 tiles from openstreetmap.org logs at 512 pixel zoom. NOTE: all other zooms in this document use nominal 256 pixel zooms, offset by 1)._
+![tilezen_size_v1d5_versus_v1d6](https://user-images.githubusercontent.com/853051/50316832-3cd43f80-046d-11e9-855e-5c2e0487dc48.gif)
```

Comment:
Since this in the `CHANGELOG.md`, it feels like we should also check in the comparison image. Having it hosted on Github is no problem, but it feels neater to me if the repo is self-contained?